androidshared/src/main/res/values/screen_names.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I feel like this should live in `collect_app` as it's about screens that are implemented there.

Reply:
Done.